iree/compiler/Dialect/VM/Conversion/UtilToVM/ConvertAlignmentOps.cpp

```diff
@@ -0,0 +1,85 @@
+// Copyright 2021 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree/compiler/Dialect/Util/IR/UtilOps.h"
+#include "iree/compiler/Dialect/Util/IR/UtilTypes.h"
+#include "iree/compiler/Dialect/VM/Conversion/TypeConverter.h"
+#include "iree/compiler/Dialect/VM/Conversion/UtilToVM/ConvertUtilToVM.h"
+#include "iree/compiler/Dialect/VM/IR/VMOps.h"
+#include "mlir/IR/Attributes.h"
+#include "mlir/IR/Builders.h"
+#include "mlir/IR/BuiltinOps.h"
+#include "mlir/IR/Matchers.h"
+#include "mlir/Transforms/DialectConversion.h"
+
+namespace mlir {
+namespace iree_compiler {
+namespace {
+
+template <typename CONST_OP, typename SUB_OP, typename ADD_OP, typename NOT_OP,
+          typename AND_OP>
+void insertAlignOps(IREE::Util::AlignOp srcOp,
+                    ConversionPatternRewriter &rewriter,
+                    ArrayRef<Value> operands, Location opLoc,
```

Comment:
I'd drop the `opLoc` parameter and just grab it from `srcOp.getLoc()` within this function. Less room for error that way (since as you have it now you _could_ call this method with a different `Location` when you probably don't want to)